lms/templates/index.html

```diff
@@ -179,6 +179,83 @@ <h2 class="header-search-facets">${_('Refine Your Search')}</h2>
+            "course-v1:LakesideGlobalInstitute+2000+2020_T1",  // Enhancing Trauma Awareness
+            "course-v1:Lakeside+PRE1101+2020",  // PRESENCE - Introductory Track
+            "course-v1:Lakeside+PRE1101+2020_T2",  // PRESENCE-Leadership Track
+            "course-v1:Lakeside+PRE1103+2029_T2",  // PRESENCE - Clinical Track
+          ]
+
+          for (let el of $(".courses-listing")) {
+
+            const firstCoursesHtml = [];
+            const remainingCoursesHtml = [];
+            let allCourses = [];
+
+            const getCourseKey = function (element) {
+              // e.g. "/courses/course-v1:Lakeside+PRE1103+2029_T2/about"
+              let href = element.childNodes[1].childNodes[1].attributes["href"];
+              if (typeof (href) === "object") {
+                // Not sure why object in some cases
+                href = String(href.value);
+              }
+              let courseKey = null;
+              const courseKeyMatch = href.match(/(?<=courses\/)(.*?)(?=\/)/);
+              if (courseKeyMatch === null) {
+                // Might not be a course (a product, probably).
+                // Shouldn't run into this in the courses-listing section of interest.
+                return;
+              } else {
+                courseKey = courseKeyMatch[0];
+              }
+              // e.g. "course-v1:Lakeside+PRE1103+2029_T2"
+              return courseKey;
+            }
+
+            // Add all the courses from the current courses-listing, excluding the ones configured in COURSES_ORDER
+            for (let i = 0; i < el.children.length; i++) {
+              const li = el.children[i];
+              const courseKey = getCourseKey(li);
+              if (COURSES_ORDER.indexOf(courseKey) === -1) {
+                remainingCoursesHtml.push("<li class='courses-listing-item' test-attr='attr'>" + li.innerHTML + "</li>");
+              }
+            }
+
+            // Populate the first batch of courses (the one we want sorted as per COURSES_ORDER)
+            for (let k = 0; k < COURSES_ORDER.length; k++) {
+              for (let i = 0; i < el.children.length; i++) {
+                const li = el.children[i];
+                const courseKey = getCourseKey(li);
+                if (courseKey === COURSES_ORDER[k]) {
+                  firstCoursesHtml.push("<li class='courses-listing-item' test-attr='attr'>" + li.innerHTML + "</li>");
+                }
+              }
+
+            }
+
+            // Replace the courses-listing section's content with a new one
+            if (firstCoursesHtml.length > 0) {
+              allCourses = firstCoursesHtml.concat(remainingCoursesHtml);
+              el.innerHTML = allCourses.join("");
+            } else {
+              console.info("Didn't sort a certain group of courses, and it's fine.")
+            }
+
+          }
+        }
+
+        setTimeout(reorderCourses, 3000);
```

Comment:
3000 is for really slow connection

setTimeout is to wait for all other scripts to execute

Reply:
let's do 1000, it will be enough, as all users are from the North America

Reply:
addressed